### DISCUSSION TITLE
graphql/scalars/docs/scalars/positive-int

### DISCUSSION THREAD

**Discussion by giscus** [Urigo/graphql-scalars#2380] [Docs Discussions]
# graphql/scalars/docs/scalars/positive-int

https://the-guild.dev/graphql/scalars/docs/scalars/positive-int

**Comment by killjoy2013** [comment]
Is it possible to add constraint? Like, positive integer smaller than 10....

**Reply by ardatan** [reply]
You should create your own scalar for that constraint. Currently any of these scalars support this kind of extra constraints.
